docs/application/dotnet/guides/multimedia/image-edit.md

```diff
@@ -0,0 +1,343 @@
+# Image Editing
+
+
+Tizen offers various image processing features.
+
+The main features of the ImageUtil API include the following:
+
+- Converting
+
+  You can change the [image color space](#image_colorspace) among the [supported formats](#color_format).
+
+- Resizing
+
+  You can change the [image resolution](#resize_image).
+
+- Rotating
+
+  You can change the [image angle](#rotate_image) around the X or Y axis.
+
+- Cropping
+
+  You can [crop the outer parts of the image](#crop_image).
+
+- Decoding or encoding from a file or memory and encoding to a file or memory
+
+  You can [decode images](#decode) and [encode them](#encode) with the following formats:
+
+  - Bitmap formats: YUV420, YUV422, RGB888, RGBA8888, BGRA8888, and ARGB8888
+  - Input image formats for decoding: JPEG, PNG, GIF, BMP, WEBP, HEIF, and JPEG XL
+  - Output image formats for encoding: JPEG, PNG, GIF, BMP, WEBP, and JPEG XL
+
+  > [!NOTE]
+  > You must pay attention to how the [image quality depends on the size](#quality) and compression ratio.
+
+## Prerequisites
+
+To enable your application to use the image util functionality, follow these steps:
+
+1. To find out the color spaces supported on the device, use `ImageUtil.GetSupportedColorSpaces()`:
+
+   ```csharp
+   foreach (var colorSpace in ImageUtil.GetSupportedColorSpaces(ImageFormat.Jpeg))
+   {
+      ...
+   }
+
+   or
+
+   var colorSpace = ImageUtil.GetSupportedColorSpaces(ImageFormat.Jpeg);
+   ```
+
+   The supported color space will be one of the `Tizen.Multimedia.ColorSpace` enumeration.
+
+2. To transform image, create `ImageTransformer` and `MediaPacket` for image source:
+   ```csharp
+   var imageTransformer = new ImageTransformer();
+   var packet = MediaPacket.Create(new VideoMediaFormat(MediaFormatVideoMimeType.I420, 640, 480));
+   ```
+
+<a name="image_colorspace"></a>
+## Convert image color space
+
+To convert one color space of an image to another, follow these steps:
+
+1. Create a `ColorSpaceTransform` instance:
+
+   ```csharp
+   var colorSpaceTransform = new ColorSpaceTransform(ColorSpace.I420);
+   ```
+
+   or, the color space of target image can be set by `ColorSpace` property after `ColorSpaceTransform` is created.
+
+   ```csharp
+   colorSpaceTransform.ColorSpace = ColorSpace.NV12;
+   ```
+
+2. Execute the transformation using `TransformAsync()` of `ImageTransformer` class:
```

Comment:
Link ImageTransformer (https://docs1.stg.tizen.org/application/dotnet/api/TizenFX/latest/api/Tizen.Multimedia.Util.ImageTransformer.html)

Reply:
I'll add the link for ImageTransformaer at line 54.